### PR TITLE
feat: notifications

### PR DESCRIPTION
**Pull request by InnaAtanasova**
## Related Issue
Closes SAP/fundamental-styles#289

## Description
This is a new component. 
Notifications have 3 sizes: default, medium and small
The types of the notifications are: default, success, error, warning, and information.
There is a vertical stripe at the left hand side of the notification representing the Priority Indicator. An optional semantic icon can be added in the Title area.

### CODE REVIEW

File: docs/_data/versions.yml [hunk 1]
```diff
@@ -190,6 +190,10 @@ multi-input:
   since: 1.1.0
   updated: 1.2.1
 
+notification:
+  since: 1.1.0
+  updated:
+
```
Comment (jbadan) [inline docs/_data/versions.yml]: I think we got rid of this file

**Comment by jbadan** [comment]
@InnaAtanasova Can you merge master into your branch so it's updated with latest. Only other thing to do with a new component is to add an import to this file: https://github.com/SAP/fundamental-styles/blob/master/src/fundamental-styles.scss

File: docs/_data/versions.yml (hunk 1 above)
Comment (droshev) [reply docs/_data/versions.yml]: done

File: scss/components/notification.scss [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+
+    // ------------------------ HEADER ------------------------
+    &__header {
+        @include fd-reset();
+        display: flex;
+        box-shadow: inset 0 -0.0625rem var(--sapPageHeader_BorderColor, $sapPageHeader_BorderColor);
+    }
+
+    &__indicator {
```
Comment (jbadan) [inline scss/components/notification.scss]: I think we need some sizing resets here:
Screen Shot 2019-09-11 at 3 14 23 PM

File: test/templates/notification/index.njk [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+<div class="fd-notification fd-notification--success">
+    <div class="fd-notification__header">
+        <div class="fd-notification__indicator--success"></div>
+        <div class="fd-notification__title">This is the notifications title Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</div>
+        <button class="btn-playground fd-notification__close"></button>
+    </div>
+    <div class="fd-notification__body">
+        <div class="fd-notification__content">
+            <div class="fd-notification__avatar">
+                <span>WW</span>
```
Comment (jbadan) [inline test/templates/notification/index.njk]: If you put a placeholder class on this -- like one of the ones on `layout.css`, it won't throw you off on the unnormalize checkbox.

**Comment by jbadan** [comment]
Other than those two minor things I'm on board! Looks great 🎉

File: test/templates/notification/index.njk (hunk 3 above)
Comment (droshev) [reply test/templates/notification/index.njk]: done

File: scss/components/notification.scss (hunk 2 above)
Comment (droshev) [reply scss/components/notification.scss]: done

File: docs/_data/components.yml [hunk 4]
```diff
@@ -45,6 +45,12 @@
     - title: Modal
       meta: modal
 
+    - title: Multi Input
```
Comment (jbadan) [inline docs/_data/components.yml]: Should multi input be here? I thought it moved ot patterns.

File: scss/_settings.scss [hunk 5]
```diff
@@ -1,6 +1,8 @@
 //library namespace
 $fd-namespace: fd !default;
 
+@import "./../node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/sass_variables.scss";
```
Comment (jbadan) [inline scss/_settings.scss]: are we taking the sass variables or the css variables? Should this also be passed to the postcss config? I think there's some infrastructure concerns here. 

I'm ok with this as long as someone has checked that the postcss color backups still work as expected.

File: docs/_data/components.yml (hunk 4 above)
Comment (jbadan) [reply docs/_data/components.yml]: Nevermind it should be!

File: scss/_settings.scss (hunk 5 above)
Comment (droshev) [reply scss/_settings.scss]: i thought we agreed to pass both(css and scss variables), if this is not the case we can clean it up

Comment (jbadan) [reply scss/_settings.scss]: No that's fine, but did we check this works as expected with postcss?

You'll need to check the docs site as the playground isn't wired up to have the scss run through postcss, see: https://github.com/SAP/fundamental-styles/issues/296

File: scss/components/notification.scss [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+    $fd-notification-indicator-color-success: var(--sapSuccessColor, $sapSuccessColor) !default;
+    $fd-notification-indicator-color-information: var(--sapInformationColor, $sapInformationColor) !default;
+
+    $fd-notification-indicator-border-color-error: var(--sapErrorBorderColor, $sapErrorBorderColor) !default;
+    $fd-notification-indicator-border-color-warning: var(--sapWarningBorderColor, $sapWarningBorderColor) !default;
+    $fd-notification-indicator-border-color-success: var(--sapSuccessBorderColor, $sapSuccessBorderColor) !default;
+    $fd-notification-indicator-border-color-information: var(--sapInformationBorderColor, $sapInformationBorderColor) !default;
+
+    $fd-notification-title-color: var(--sapGroup_TitleTextColor, $sapGroup_TitleTextColor) !default;
+    $fd-notification-content-text-color: var(--sapContent_LabelColor, $sapContent_LabelColor) !default;
```
Comment (jbadan) [inline scss/components/notification.scss]: so I think if we setup postcss correctly, none of this is necessary. You would just need `var(--sapContent_LabelColor)` and that's it.

**Pull request closed without merging**